lib/ci_helper/commands/check_spec_suffixes.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module CIHelper
+  module Commands
+    class CheckSpecSuffixes < BaseCommand
+      def call
+        target_paths.reject { |path| path.end_with?("_spec.rb") }.tap do |paths|
```

Comment:
Why do we use `#tap` method here?
We can't just assign to the var?

```ruby
paths = target_paths.reject { |path| path.end_with?("_spec.rb") }
```

Reply:
tap ftw